.azure-pipelines/functionality-test.yml

```diff
@@ -19,13 +19,6 @@ variables:
   value: $(Build.BuildNumber)
```

Comment:
For variables start with `TEST_`, shouldn't they change to read from environment variable so we can control behavior in build?